qvm/sys-gui-gpu-attach-gpu.sls

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# qvm.sys-gui-gpu-vm-attach-gpu
+# ===========
+##
+
+sys-gui-gpu-attach-gpu:
+  qvm.devices:
+    - name: sys-gui-gpu
+    - attach:
+      - pci:dom0:00_02.0:
+        - permissive: true
+
+sys-gui-gpu-enable-gfx-passthru:
+  qvm.features:
+    - name: sys-gui-gpu
+    - enable: 
+      - gfx-passthru
```

Comment:
This part is now unnecessary (https://github.com/QubesOS/qubes-core-libvirt/pull/23#issuecomment-640141610).